src/datepicker/bs-datepicker.module.ts

```diff
@@ -34,6 +32,13 @@ import { BsDatepickerDayDecoratorComponent } from './themes/bs/bs-datepicker-day
     BsDatepickerComponent, BsDaterangepickerComponent]
 })
 export class BsDatepickerModule {
+  constructor() {
+    if (isDevMode()) {
+      console.warn(`BsDatepickerModule is under development,
+      BREAKING CHANGES are possible,
+      PLEASE, read changelog`);
+    }
+  }
```

Comment:
I guess we should exclude this message from tests

![image](https://user-images.githubusercontent.com/75517/30069121-16f59672-9260-11e7-8ef6-0803dcf464af.png)

Reply:
question is how to figure out that we are under testing?
what I can do is to show this warning only once (as was initially planned):
https://github.com/valor-software/ngx-bootstrap/pull/2585